### ISSUE TITLE
Getting started: No module named 'survey' error

### ISSUE THREAD

**Issue by nvladimus** [Pierre-Sassoulas/django-survey#126]
Hi, everyone,
I am a beginner in Django, and would like to try the `survey` app. I followed instructions in `Getting started` (installed with pip, changed `settings.py` and `urls.py` in my project), but when I try to launch `python manage.py runserver`, I get the following errors:
```
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\threading.py", line 916, in _bootstrap_inner
    self.run()
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\utils\autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\core\management\commands\runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\utils\autoreload.py", line 76, in raise_last_exception
    raise _exception[1]
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\core\management\__init__.py", line 357, in execute
    autoreload.check_errors(django.setup)()
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\utils\autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\apps\registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\site-packages\django\apps\config.py", line 90, in create
    module = import_module(entry)
  File "C:\Users\username\AppData\Local\Continuum\Anaconda2\envs\py36\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'survey'
```
My parent project runs fine without the `survey` app. I am using Python 3.6 and Django 3.0.8. Could anyone please give some hints how to import the `survey` app? Totally stuck on step 1...
Many thanks!

**Comment by Pierre-Sassoulas** [comment]
Hello, it look like the package is not installed or not found.  It looks like you're in an anaconda environment, the package is not available using conda forge, you need to use pip to install it. What output does the command `pip install django-survey-and-report` or `pip3 install django-survey-and-report` gives you ?

**Comment by nvladimus** [comment]
Thanks for quick reply! It turned out I had `PyYAML` of older version which was causing conflict:
```
Attempting uninstall: pyyaml
    Found existing installation: PyYAML 3.12
ERROR: Cannot uninstall 'PyYAML'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
```
I ran `pip install --ignore-installed PyYAML` and then `pip install django-survey-and-report`, which did the job.
My project now launches successfully!
Another newbie question: how can I launch the `survey` app within my project? The `Getting started` does not suggest that I run any `python manage.py makemigrations` before using it. Can I launch it without migrating my database? I tried url `http://127.0.0.1:8000/^survey/`, but got 404 error. I realize these questions are very dumb for Django people, but this is really my first project apart from tutorials.
Thank you!
